run_unittests.py

```diff
@@ -381,6 +381,8 @@ def _run(self, command, workdir=None):
         output = p.communicate()[0]
         print(output)
         if p.returncode != 0:
+            if 'MESON_SKIP_TEST' in output:
+                raise unittest.SkipTest('Project requested skipping.')
```

Comment:
I'm not talking about the unit test, I'm talking about the specific project test that requested a skip. You could look at the unit test code and see what project test it was using, but that is not straightforward for a lay user, and a unit test may use more than one project test anyway.